routes/socket/game/end-game.js

```diff
@@ -179,20 +179,18 @@ module.exports.completeGame = (game, winningTeamName) => {
 	game.general.isRecorded = true;
 
 	// Don't compute Elo for private, casual, custom, private, or unlisted games
-	if (
-		!game.general.private &&
-		!game.general.casualGame &&
-		!(game.customGameSettings && game.customGameSettings.enabled) &&
-		!game.general.practiceGame &&
-		!game.general.unlisted
-	) {
+	if (!game.general.private && !game.general.casualGame && !(game.customGameSettings && game.customGameSettings.enabled) && !game.general.unlisted) {
 		Account.find({
 			username: { $in: seatedPlayers.map(player => player.userName) }
 		})
 			.then(results => {
 				const isRainbow = game.general.rainbowgame;
 				const isTournamentFinalGame = game.general.isTourny && game.general.tournyInfo.round === 2;
-				const eloAdjustments = rateEloGame(game, results, winningPlayerNames);
+				let eloAdjustments;
+
+				if (!game.general.practiceGame) eloAdjustments = rateEloGame(game, results, winningPlayerNames);
+
+				const practiceSuffix = game.general.practiceGame ? 'Practice' : '';

```

Comment:
`practiceSuffix` feels a bit clunky.

Instead of doing things this way, why not log the total games played that could count for rainbow as `account.gamesPlayed{SEASON}`? You would just increment this after each ranked/practice game played. If you want to let at most 20 practice games count for rainbow, you can not increment `gamesPlayed` if `gamesPlayed - losses - wins >= 20`. This makes the rainbow formula a lot simpler, since you'd then just need to check that `gamesPlayed >= 50`.

Just my two cents.